wiki/Tournaments/Mirai/2020/en.md

```diff
@@ -0,0 +1,570 @@
+| Designer     | ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Referee      | ![][flag_gb] [AlexDark69](https://osu.ppy.sh/users/5146647), ![][flag_de] [GDLenny](https://osu.ppy.sh/users/8406711), ![][flag_de] [Inteleon](https://osu.ppy.sh/users/9665028), ![][flag_tr] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_nl] [Mirai Subject](https://osu.ppy.sh/users/5639709), ![][flag_vn] [rock-on](https://osu.ppy.sh/users/9676089), ![][flag_ph] [Simon Yuusuke](https://osu.ppy.sh/users/7336035), ![][flag_ca] [Takiaz](https://osu.ppy.sh/users/11177233), ![][flag_ca] [TheHunter1](https://osu.ppy.sh/users/6496016) |
+| Statistician | ![][flag_ph] [kristoffer09](https://osu.ppy.sh/users/3474344)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1047235)
+- [Challonge brackets](https://challonge.com/Mirai2020)
+- [Mirai. Community Discord Server](https://discord.gg/cSfhfm2)
+- [Livestream](https://www.twitch.tv/MiraiSubject)
+
+## Participants
+
+|          Team          | Members                                                                                                                                                                                            |
+| :--------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|   **8 min tech map**   | [poboi06](https://osu.ppy.sh/users/12836907), [SirPinky](https://osu.ppy.sh/users/12948136), [meapii](https://osu.ppy.sh/users/12643092), [His](https://osu.ppy.sh/users/5433421)                  |
```

Comment:
```suggestion
| **8 min tech map** | [poboi06](https://osu.ppy.sh/users/12836907), [SirPinky](https://osu.ppy.sh/users/12948136), [meapii](https://osu.ppy.sh/users/12643092), [His](https://osu.ppy.sh/users/5433421) |
```
Remove unnecessary whitespaces. You only need one padding in the end and beginning of a content.
And if you can, stick player's flag